### PR TITLE
Multichannel

### PR DESCRIPTION
**Pull request by happyhuman**
Samples to demo multi channels

### CODE REVIEW

**Comment by happyhuman** [comment]
@nnegrey , @dizcology , @theacodes Can you please review this PR?

File: speech/cloud-client/transcribe_multichannel.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    with open(speech_file, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = speech.types.RecognitionAudio(content=content)
+
+    config = speech.types.RecognitionConfig(
+        encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='en-US',
+        audio_channel_count=1,
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multichannel.py]: Should we find a audio file that has 2 channels to show the difference?

File: speech/cloud-client/transcribe_multichannel_test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+
+import transcribe_multichannel
+
+RESOURCES = os.path.join(os.path.dirname(__file__), 'resources')
+OUTPUT1 = 'OK Google stream stranger things from Netflix to my TV'
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multichannel_test.py]: NIT: Remove the 1 on OUTPUT1 if there is only 1 output.

File: speech/cloud-client/transcribe_multichannel.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+# limitations under the License.
+
+r"""Google Cloud Speech API sample that demonstrates how to use multiple
+audio channels.
+
+Example usage:
+    python transcribe_multichannel.py \
+        resources/Google_Gnome.wav
+    python transcribe_multichannel.py \
+        gs://clud-ml-api-e2e-testing/speech/stereo_audio.wav
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multichannel.py]: This a public bucket? `clud-ml-api-e2e-testing`

**Comment by dizcology** [comment]
Why is this closed without merge?

**Pull request closed without merging**